python/src/trezorlib/cli/ethereum.py

```diff
@@ -74,6 +75,25 @@ def _amount_to_int(ctx, param, value):
         raise click.BadParameter("Amount not understood")
 
 
+def _parse_access_list(ctx, param, value):
+    try:
+        return [_parse_access_list_item(ctx, param, val) for val in value]
+
+    except Exception:
+        raise click.BadParameter("Access List format invalid")
+
+
+def _parse_access_list_item(ctx, param, value):
```

Comment:
this probably doesn't need the `ctx, param` arguments?

Reply:
Nope, fixed!